### PR TITLE
troubleshooting.md: added #19 not enough ids

### PR DESCRIPTION
**Pull request by nitrocode**
https://github.com/containers/libpod/issues/3421#issuecomment-544455837

### CODE REVIEW

File: troubleshooting.md [hunk 1]
```diff
@@ -429,3 +429,28 @@ You'll need to either:
...
+
+```console
+$ podman unshare cat /proc/self/uid_map
+         0       1000          1
+```
+
+#### Solution
+
+```console
+$ podman system migrate
```
Comment (giuseppe) [inline troubleshooting.md]: this won' be enough if there are no IDs added for the user.

I think we should document that additional IDs must be provided through the `/etc/subuid` and `/etc/subgid` files.

**Comment by rhatdan** [comment]
@nitrocode you need to sign your PRs.

git commit -a --amend -s
git push --force

File: troubleshooting.md (hunk 1 above)
Comment (nitrocode) [reply troubleshooting.md]: What additional content would you recommend? I can provide the outputs of those files at the very least.

```
$ cat /etc/subgid 
user:100000:65536
$ cat /etc/subuid
user:100000:65536
```

Comment (giuseppe) [reply troubleshooting.md]: I think we can add the example you just showed, and also warn the user about overlapping ranges.  The IDs specified in these file must be unique for each user.

I'd also add a pointer to the `subuid(5)` and `subgid(5)` man pages.

File: troubleshooting.md [hunk 2]
```diff
@@ -429,3 +429,28 @@ You'll need to either:
 or as root if your user has not enough privileges.
 
 * sudo loginctl enable-linger $UID
+
+### 18) error creating libpod runtime: there might not be enough IDs available in the namespace
```
Comment (TomSweeneyRedHat) [inline troubleshooting.md]: #4614 just beat you to the punch with a separate update.  Please rebase and change this to `### 19)`

**Comment by TomSweeneyRedHat** [comment]
Looks like you need to rebase still @nitrocode

**Comment by nitrocode** [comment]
I bumped #18 to #19 and added links to subuid and subgid man pages.

**Comment by TomSweeneyRedHat** [comment]
@nitrocode were you able to sign your commit?  @giuseppe do the new changes cover your concerns?

**Comment by nitrocode** [comment]
Aw I forgot. I'll sign them again tonight when I'm home.

**Comment by nitrocode** [comment]
Sorry for the delay. I signed the commit and force pushed.

**Comment by TomSweeneyRedHat** [comment]
Thanks @nitrocode .   @giuseppe PTAL and merge if it LGTY
LGTM

**Pull request merged**